### PR TITLE
fixed compiler pass, added mailer command

### PR DESCRIPTION
**Pull request by pulse00**
There was an issue in my last PR, this one fixes it. Sorry about that.

Also this PR includes a command to send test mails.

### CODE REVIEW

File: Command/SendEmailCommand.php [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $to = $input->getArgument('to');
+        $from = $input->getArgument('from');
+        $validator = $this->getContainer()->get('validator');
+
+        $constraints = array(
+            new \Symfony\Component\Validator\Constraints\Email(),
```
Comment (henrikbjorn) [inline Command/SendEmailCommand.php]: Should be added as use statements.

File: Command/SendEmailCommand.php [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        }
+
+        /** @var MailerInterface $mailer */
+        $mailer = $this->getContainer()->get('hb_stampie.mailer');
+        $mailerClass = new \ReflectionClass($mailer);
+
+        $output->writeln(sprintf('Sending message from <info>%s</info> to <info>%s</info> using <info>%s</info> mailer',
+            $from, $to, $mailerClass->getShortName()));
+
+        $identity= new \Stampie\Identity($to);
```
Comment (henrikbjorn) [inline Command/SendEmailCommand.php]: use statement.

File: Command/SendEmailCommand.php [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+class SendEmailCommand extends ContainerAwareCommand
+{
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('hb:stampie:send')
```
Comment (henrikbjorn) [inline Command/SendEmailCommand.php]: just stampie:send would be ok :) or stampie:test

File: Command/SendEmailCommand.php [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class TestMessage
+ * @package HB\StampieBundle\Command
+ */
+class TestMessage extends \Stampie\Message
```
Comment (henrikbjorn) [inline Command/SendEmailCommand.php]: split into its own file.

**Comment by pulse00** [comment]
@henrikbjorn changes pushed.

**Comment by henrikbjorn** [comment]
Thanks

**Comment by Ninir** [comment]
Hey guys, could the fix be cherry-pick and tagged so that we can use the stable version with the fixed Guzzle support? :)

**Comment by Ninir** [comment]
ping @henrikbjorn

**Comment by henrikbjorn** [comment]
@Ninir sure. Which exact commits do you want cherry picked? or just the whole PR?

**Comment by Ninir** [comment]
I would say this PR if you agree?

**Pull request merged**